rudder-agent/debian/dirs

```diff
@@ -7,7 +7,4 @@ var/rudder/cfengine-community/inputs
 var/rudder/cfengine-community/bin
 opt/rudder/etc
 var/rudder/tmp
-var/rudder/tools
```

Comment:
because dirs is for directort creation and this creation is already done in the makefile